ui/tests/integration/task-log-test.js

```diff
@@ -8,7 +8,7 @@ import { logEncode } from '../../mirage/data/logs';
 
 const HOST = '1.1.1.1:1111';
 const commonProps = {
-  interval: 50,
+  interval: 150,
```

Comment:
Intentional change?

Reply:
Intentional 😞 Basically the interval is a race against component initialization. At 50ms I thought the component was guaranteed to initialize in time, but now 50 and 100 both result in failures. It's hard to believe that these changes could have impacted performance like that, but here we are.

Reply:
Ah, I didn't realize that's what it was for. In case you hadn't considered it, maybe [registering a custom waiter would make this test more reliable](https://medium.com/@chrisdmasters/testing-async-in-ember-js-part-one-7ec9bc070c0e)?

Reply:
I did some more science around the interval and it isn't render time. It seems to be just general computation cycles, which makes it especially hard to get around the issue with a custom waiter. This is why I am always disappointed in timer based assertions, but without better tooling, this is the only choice.

The good news is I couldn't replicate the issue where it was consistently failing at 50ms, so I'm going to revert this change.